vector/src/main/res/values/strings.xml

```diff
@@ -410,10 +413,18 @@
     <string name="room_participants_action_devices_list">Show Device List</string>
     <string name="room_participants_power_level_prompt">You will not be able to undo this change as you are promoting the user to have the same power level as yourself.\nAre you sure?</string>
 
-    <string name="room_participants_invite_prompt_msg">"Are you sure you want to invite %s to this chat?"</string>
     <string name="room_participants_kick_prompt_msg">Are you sure that you want to kick this user from this chat?</string>
     <string name="room_participants_ban_prompt_msg">Are you sure that you want to ban this user from this chat?</string>
 
+    <!-- The variable can be a single name or a string of names ("A, ""B, ""C and D" > "A, B, C and D) -->
```

Comment:
I do not understand the usage of `"` in this comment. Is there some typo?
Maybe write an example for clarity: `"Bob" or "Bob and Alice" or "Bob, Alice and Charlie"`

Reply:
Just to give the translator an idea how the block is built, to show that there is a space after the comma.
And yes, human names are better :)
Will work now on it.